app/src/main/java/com/incwelltechnology/lms/ui/home/DashboardActivity.kt

```diff
@@ -0,0 +1,93 @@
+package com.incwelltechnology.lms.ui.home
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import androidx.appcompat.app.ActionBarDrawerToggle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.GravityCompat
+import androidx.drawerlayout.widget.DrawerLayout
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.google.android.material.navigation.NavigationView
+import com.google.android.material.snackbar.Snackbar
+import com.incwelltechnology.lms.R
+
+class DashboardActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_dashboard)
+        val toolbar: Toolbar = findViewById(R.id.toolbar)
```

Comment:
@iamsannyrai do we need findViewById to reference element?

Reply:
> 
> 
> @iamsannyrai do we need findViewById to reference element?

No we don't need to. This activity was just created and left untouched. I haven't worked upon this activity :)